code/game/machinery/kitchen/icecream.dm

```diff
@@ -178,7 +178,8 @@
 	var/ice_creamed = 0
 	var/cone_type
 
-/obj/item/weapon/reagent_containers/food/snacks/icecream/New()
+/obj/item/weapon/reagent_containers/food/snacks/icecream/Initialize()
+	. = ..
```

Comment:
Bad return.

Reply:
Read your mind and fixed it before you made the review

Reply:
😠